packages/web3torrent/src/pages/file/File.tsx

```diff
@@ -9,8 +9,21 @@ import {parseURL, useQuery} from '../../utils/magnet';
 import {torrentStatusChecker} from '../../utils/torrent-status-checker';
 import {useInterval} from '../../utils/useInterval';
 import './File.scss';
-import WebTorrentPaidStreamingClient from '../../library/web3torrent-lib';
+import WebTorrentPaidStreamingClient, {TorrentTestResult} from '../../library/web3torrent-lib';
 import _ from 'lodash';
+import {Flash} from 'rimble-ui';
+
+const checkTorrent = async (web3Torrent, infoHash) => {
```

Comment:
Use function syntax and add types to parameters.

Reply:
https://github.com/statechannels/monorepo/pull/1449/commits/227e2312f728634016f0b2000704e71dc73c4cc1